src/TrackGenerator3D.cpp

```diff
@@ -1216,13 +1216,28 @@ void TrackGenerator3D::segmentizeExtruded() {
 
   log_printf(NORMAL, "Ray tracing for axially extruded track segmentation...");
 
-  /* Get all unique z-coords at which 2D radial segementation is performed */
+  /* Get all unique z-coords at which 2D radial segmentation is performed */
   std::vector<double> z_coords;
   if (_contains_segmentation_heights)
     z_coords = _segmentation_heights;
   else
     z_coords = _geometry->getUniqueZPlanes();
 
+  /* Print number of unique Z planes */
+  int num_Z_planes = z_coords.size();
+  int _num_Z_planes = num_Z_planes - 1;
+  int max_num_Z_planes = num_Z_planes;
+#ifdef MPIx
+  if (_geometry->isDomainDecomposed()) {
+    MPI_Allreduce(&num_Z_planes, &_num_Z_planes, 1, MPI_INT, MPI_SUM,
+                  _geometry->getMPICart());
+    MPI_Allreduce(&num_Z_planes, &max_num_Z_planes, 1, MPI_INT, MPI_MAX,
+                  _geometry->getMPICart());
+  }
+#endif
+  log_printf(NORMAL, "Max / total number of unique Z-planes %d / %d",
+             max_num_Z_planes, _num_Z_planes + 1);
```

Comment:
The total doesn't work that way. It'd be more code that it's worth to report it properly. I think the max is still somewhat relevant, the log could stay as INFO. What do you think ?

Reply:
Any necessity for this output? And I don't quite understand what is the meaning of this code snap

Reply:
The total number of Z-planes is somewhat interesting to the user. The fewer Z-planes the better for speed. 
Deleting this code because it doesn't provide that.